egov-persister/auth-sek-session-persister.yml

```
@@ -49,3 +49,22 @@ serviceMaps:
               - jsonPath: $.TreasuryPaymentData.remarkStatus
               - jsonPath: $.TreasuryPaymentData.remarks
               - jsonPath: $.TreasuryPaymentData.fileStoreId
+
+    - version: 1.0
+      description: Treasury head breakup mapping
+      fromTopic: create-treasury-mapping
+      objecIdJsonPath: $.consumerCode
+      tenantIdJsonPath: $.tenantId
+      queryMaps:
+          - query: INSERT INTO treasury_head_breakup_data (consumer_code, head_mapping,calculation, tenant_id, createdtime) values(?, ?, ?, ?, ?)
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 10

(indentation)